spec/ttml1.xml

```diff
@@ -7469,27 +7469,25 @@ associated (owning) element node.</p>
 <gitem>
 <label>[associate region]</label>
 <def>
-<p>A <loc href="#element-vocab-type-content">Content</loc> element is associated with a region according
-to the following ordered rules, where the first rule satisfied
-is used and remaining rules are skipped:</p>
+<p>A <loc href="#element-vocab-type-content">Content</loc> element <emph>E</emph> is associated 
+with a region <emph>R</emph> if
+any of the following ordered assertions is true, where the first assertion satisfied is used and remaining assertions are skipped:</p>
 <olist>
-<item><p>if the element specifies a <loc
-href="#layout-attribute-region"><att>region</att></loc> attribute,
-then the element is associated with the region referenced by that
-attribute;</p></item>
-<item><p>if some ancestor of that element specifies a <loc
-href="#layout-attribute-region"><att>region</att></loc> attribute,
-then the element is associated with the region referenced by the most
-immediate ancestor that specifies this attribute;</p></item>
-<item><p> if the element contains a descendant element that
-specifies a <loc
-href="#layout-attribute-region"><att>region</att></loc> attribute,
-then the element is associated with the region referenced by that
-attribute;</p></item>
+
+<item><p>if <emph>E</emph> specifies a <loc href="#layout-attribute-region">
+<att>region</att></loc> attribute that references <emph>R</emph>;</p></item>
```

Comment:
My reading of this condition is that if E specifies a region attribute that references a region _other than_ R then this condition is skipped and the next is evaluated. However the introduction of the "stopping" rule at line 7474 makes me wonder if in that case the intent is to stop and return false, since the condition "E specifies a region attribute" is true, but the region attribute that it specifies is different. If that is the intent then this needs to be reworded. Either way it would be useful to clarify.